tox.ini

```diff
@@ -1,7 +1,10 @@
 #
 # Configuration for tox and pytest
 
+# first five ignores are defaults per http://flake8.readthedocs.io/en/latest/config.html#default
 [flake8]
+ignore = E123, E133, E226, E241, E242, W503, E402
+max-line-length = 77
 exclude = dist,docs,*.egg-info,.git,ref,_scratch,.tox
```

Comment:
The flake8 limit should be 79 by default. The 77 limit is just a project convention, for uncommented code, so that if you *do* comment it out it doesn't go past the 79 limit. So I don't see a way to enforce it with flake, I just use the textwidth in vim to wrap it appropriately while editing.